Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -455,6 +455,7 @@ public class GraphLinter: GraphLinting {
             LintableTarget(platform: .iOS, product: .bundle),
             LintableTarget(platform: .iOS, product: .appClip),
             LintableTarget(platform: .macOS, product: .macro),
+            LintableTarget(platform: .iOS, product: .appExtension),
```

Comment:
Can we test this combination in `GraphLinterTests`?

Reply:
Added test